content/blog/posts/2022-05-19-five-tips-to-improve-your-terminal-experience.md

```diff
@@ -0,0 +1,70 @@
+---
+authorId: rwwagner90
+categories:
+  - devtools
+date: '2022-05-19'
+slug: five-tips-to-improve-your-terminal-experience
+title: 5 Tips to Improve Your Terminal Experience
+---
+
+We've come a long way from the stereotypical terminals seen in the movies with
```

Comment:
Wondering if the next 2 comments could add some more clarity to the intro?

"Working in software, we know the stereotypical terminals shown in movie scenes - with green text whizzing through black backgrounds as a hacker furiously slings commands from memory onto an eight inch screen - is far from the reality of development."

Reply:
Ohhh this sounds so much better!